lib/LANraragi/Model/Tankoubon.pm

```diff
@@ -419,4 +490,40 @@ sub get_tankoubons_containing_archive ($arcid) {
     return @tankoubons;
 }
 
+sub update_metadata_field ( $tank_id, $field, $value ) {
+    my $redis        = LANraragi::Model::Config->get_redis;
+    my $logger       = get_logger( "Tankoubon", "lanraragi" );
+    my $err          = "";
```

Comment:
This variable does nothing here now, right?  
There's no error checking at all here so you don't even need a return statement at this point it seems..